ParisBNF/abb/BNFabb39.xml

```diff
@@ -432,28 +438,31 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                             in Gǝʿǝz, in some cases in Amharic (<locus target="90ra"/>); the units are
                                               separated mostly by ፤ sign. </note>
                                             </msItem>
+                                  </msItem>
 
-                                            <msItem xml:id="p1_i5">
-                                              <locus from="109va" to="203vb"/>
-                                              <title type="complete" ref="LIT2293Sawase">Another collection of commentary and lexical treatise
-                                                 for various books </title>
-
-                                                <msItem xml:id="p1_i5.1">
+                                            
+                                                <msItem xml:id="p1_i5">
                                                <locus from="109va" to="110rb"/>
-                                               <title type="complete">Explanations for measures of weight in Hebrew</title>
-                                               <textLang mainLang="gez" otherLangs="am"></textLang>
+                                                  <title type="complete" ref="LIT5810ExplanationHebrewNames" cert="medium">Explanations for measures of weight in Hebrew</title>
```

Comment:
As above better: "Explanations for Hebrew weight measurements" ?

Reply:
I picked the title somewhere.

Reply:
Must be a litteral translation of a Gəʿəz title. I would suggest to change it in the LIT and in the manuscript record.

Reply:
Actually LIT5810 is for a different text, not disambiguated sufficiently. I am creating another record for measures.